packages/mui-material/src/version/index.ts

```diff
@@ -0,0 +1,25 @@
+const version = process.env.MUI_PACKAGE_VERSION || null;
+
+let versionNumber: string | null = null;
+let preReleaseInfo: string | null = null;
+
+if (version) {
+  [versionNumber, preReleaseInfo] = version.split('-');
```

Comment:
Suggestion
🤔 the parsing is still happening in the browser. An optimization could be to do this on the build side and replace multiple env variables

```tsx
export default process.env.MUI_VERSION
export const major = Number(process.env.MUI_VERSION_MAJOR)
export const minor = Number(process.env.MUI_VERSION_MINOR)
// ...
```

Reply:
Nice 👍🏼 done 😊